ui/pages/institutional/interactive-replacement-token-page/interactive-replacement-token-page.test.js

```diff
@@ -109,6 +108,18 @@ const render = ({ newState } = {}) => {
       institutionalFeatures: {
         connectRequests,
       },
+      internalAccounts: {
+        ...mockState.metamask.internalAccounts,
+        accounts: {
+          ...mockState.metamask.internalAccounts.accounts,
+          [mockState.metamask.internalAccounts.selectedAccount]: {
```

Comment:
Tricky one :) I got this right, we're populating the accounts and were are overloading the `address` of the currently selected account?

If yes, then let's add a brief comment to explain that, WDYT?